src/video/SDL_stb_c.h

```diff
@@ -27,5 +27,6 @@
 // Image conversion functions
 
 extern bool SDL_ConvertPixels_STB(int width, int height, SDL_PixelFormat src_format, SDL_Colorspace src_colorspace, SDL_PropertiesID src_properties, const void *src, int src_pitch, SDL_PixelFormat dst_format, SDL_Colorspace dst_colorspace, SDL_PropertiesID dst_properties, void *dst, int dst_pitch);
+extern bool SDL_WritePNG_STB(const char *file, SDL_Surface *surface);
```

Comment:
With an eye towards consistency with the BMP API, with an eye towards exposing it someday, maybe it should be:
```suggestion
extern bool SDL_SavePNG_IO(SDL_Surface *surface, SDL_IOStream *dst, bool closeio);
extern bool SDL_SavePNG(SDL_Surface *surface, const char *file);
```